core/ui/vapor_ui/qam/powertools_menu.gd

```diff
@@ -340,12 +369,26 @@ func _on_gpu_temp_limit_changed(value: float) -> void:
 			pass
 
 
+# Sets the thermal throttle policy for ASUS devices.
+func _on_thermal_policy_changed(index: int) -> void:
+	match index:
+			"0":
+				logger.debug("Setting thermal throttle policy to Balanced")
+			"1":
+				logger.debug("Setting thermal throttle policy to Performance")
+			"2":
+				logger.debug("Setting thermal throttle policy to Silent")
+	var args := ["setThermalPolicy", str(index)]
+	_setup_callback_exec("/sys/devices/platform/asus-nb-wmi/throttle_thermal_policy", args)
```

Comment:
Same as the other comment. We should probably not be hard coding platform specific paths here. We should be adding a function to the platform to get this path.

Reply:
as in platform.gd or ally_gen1.gd?

Reply:
You could add a property to `ally_gen1.gd`. Then add a check that the property exists.